README.md

```diff
@@ -1,64 +1,97 @@
-- On start up, show an empty page with no results
-- When the user searches for a superhero, by typing on the Search field, and hits enter, a list of 4 results will be displayed.
-- At the bottom of the page, if there are more results, a pagination system will be displayed, that will allow the user to see more results.
-
-**Optionally**, you can add the functionality that upon clicking on a specific result, the user is redirected to a detail page of that character, where you will need to show some information related to that selected character.
-
-For the information that you will be using, you will need to consume Marvel's Open API to retrieve the different characters and their information. You can take a look at the API and how to use it here:  
+For the API that the candidate will be using, the candidate will need to consume Marvel's Open API to retrieve the different characters and their information. The candidate can take a look at the API and how to use it here:  
 [https://developer.marvel.com/docs](https://developer.marvel.com/docs)
 
 You will need an API key in order to execute these requests. Please use the following one:  
 `8fb61ed0d8ae4cc76943c8d2b8e2ed9c`
 
-NB: In order to use this API key, you have to edit your `/etc/hosts` file, adding this line `127.0.0.1   develop.lumapps.com`.
+In order to use this API key, you have to edit your `/etc/hosts` file, adding this line `127.0.0.1   develop.lumapps.com`.
 
-### Mockup
+### Ground rules
 
-![App mockup](src/Frontend%20Test.png)
-
-### Rules
-
-* We strongly suggest that you use the technologies suggested under the Stack section. If you want to use another technology such as Angular JS or Vue JS because you do not know React, you can do so, but take into consideration that we do not use them in our daily basis
-* If you want to use your own boilerplate, feel free to override the whole repository.
-* Keep in mind several aspects of the application, not only functionality. Such as accessibility, performance, usability, maintainability, scalability.
-* This test should take you a day to complete, if you need more time or if you have any questions, please reach out to LumApps.
-* In order to deliver the project, please create a public GitHub repository and upload your code to it.
-* Follow the project mockups as much as you can.
+* We strongly suggest that the candidate uses the technologies suggested under the Stack section. If the candidate wants to use another technology such as Angular JS or Vue JS because they do not know React, they can do so, but they should take into consideration that we do not use them in our daily basis
+* If the candidate wants to use your own boilerplate, they are free to override the whole repository, but the candidate should note that this could take more time than just using the provided boilerplate.
```

Comment:
```suggestion
* If the candidate wants to use their own boilerplate, they are free to override the whole repository, but the candidate should note that this could take more time than just using the provided boilerplate.
```